lib/matplotlib/animation.py

```diff
@@ -895,26 +895,25 @@ class Animation:
 
     Parameters
     ----------
-    fig : `~matplotlib.figure.Figure`
-       The figure object that is used to get draw, resize, and any
-       other needed events.
+    fig : `~.Figure`
```

Comment:
It is our current policy to use fully qualified names for argument types, see https://matplotlib.org/devdocs/devel/documenting_mpl.html#referencing-types.

(multiple occurrences throught the PR)

Reply:
I don't agree with that policy, but let's put that discussion to another time.  Reverted.